lib/glossarist/related_concept.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module Glossarist
+  class RelatedConcept < Model
+    include Glossarist::Utilities::Enum
+
+    TYPES = %i[
+      deprecates
+      supersedes
+      narrower
+      broader
+      equivalent
+      compare
+      contrast
+      see
+    ]
```

Comment:
I want to be sure that we can traverse both ways, ie. Supersede, and superseded by. A relationship contains a subject, object and a relation type.